src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -527,6 +535,34 @@ describe('lambda flare', () => {
     })
   })
 
+  describe('getTags', () => {
```

Comment:
For example, here, you are never mocking back to the original value. That means that you are overwriting the previous value. What happens if you get the tags after the last test? i.e. `afterEach` get the tags again. Would the last call throw an error again? 

That's why I mentioned about the mocking on the top level and never restoring that value.

Reply:
Good point, I moved the original mock to be in an `afterAll`